File: LV0/왼쪽 오른쪽.js
function solution(str_list) {
  let firstL = str_list.indexOf("l");
  let firstR = str_list.indexOf("r");

  if (firstL === -1 && firstR === -1) {
    return [];
  } else if (firstR === -1 || (firstL !== -1 && firstL < firstR)) {
    return str_list.slice(0, firstL);
  } else {
    return str_list.slice(firstR + 1);
  }
}

File: LV0/왼쪽 오른쪽.py
def solution(str_list):
    firstL = str_list.index("l")
    firstR = str_list.index("r")

    if firstL == -1 and firstR == -1:
        return []
    elif firstR == -1 or (firstL != -1 and firstL < firstR):
        return str_list[:firstL]
    else:
        return str_list[firstR + 1:]
